source/wp-content/themes/wporg-news-2021/sass/page/front-page/_latest-posts.scss

```diff
@@ -38,11 +38,9 @@ body.news-front-page .front__latest-posts {
 		}
 
 		> li {
-			margin-bottom: calc(var(--wp--custom--alignment--edge-spacing) * 2);
+			margin-bottom: clamp( 48px, calc( 100vw / 16 ), 58px );
```

Comment:
Could that be simplified to `clamp( 48px, 6.25vw, 58px );` ?

Reply:
For sure. I was just following the way `vw` units are used elsewhere in the stylesheet. I don't feel strongly either way.

Reply:
https://github.com/WordPress/wporg-news-2021/pull/325/commits/f6a6482ee91e1211284c97eab5e1a05e685b68d8

Reply:
Yeah, I don't feel strongly either, I was just curious since the reasoning behind the longer form isn't obvious to me.

Reply:
I suspect the reasoning is just that `1vw` is basically 1/100th of the viewport width. So multiplying it by 100 gets you back to the original number that matters if you're dealing with media queries. I guess maybe it makes it easier to visualize for some people 🤷